demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -24,9 +24,9 @@
 import cv2
 import numpy as np
 
-sys.path.append(str(Path(__file__).resolve().parents[2] / 'common/python'))
+sys.path = [str(Path(__file__).resolve().parents[2] / 'common/python')] + sys.path
```

Comment:
Why do you add `str(Path(__file__).resolve().parents[2] / 'common/python')` to start?

Reply:
Please, remove the assignment here, the append returns None and it causes error while launching the demo

Reply:
Fixed.